d3a_api_client/aggregator.py

```diff
@@ -33,14 +34,15 @@ def received_message(self, message):
             self.command_response_buffer.append(message)
 
 
-class Aggregator(RestDeviceClient):
+class Aggregator(RestDeviceClient, GridFeeCalculation):
```

Comment:
I would prefer if you use composition in here , add an instance attribute `_grid_fee_calculation = GridFeeCalculation()` and then refactor accordingly .
This would save us troubles later dealing with multiple inheritance issues .
when in doubt, I always ask : 
A is  B? -> inheritance
A has B -> Composition
And we're clearly in the second case

Reply:
Ok, thanks!